Lib/concurrent/futures/process.py

```diff
@@ -253,6 +309,8 @@ def _queue_management_worker(executor_reference,
             derived from _WorkItems for processing by the process workers.
         result_queue: A ctx.SimpleQueue of _ResultItems generated by the
             process workers.
+        wakeup: A _Sentinel to allow waking up the queue_manager_thread from
+            the main Thread and avoid deadlocks caused by broken queues.
```

Comment:
I think we should rename `wakeup` to `thread_sentinel` everywhere in this PR but maybe @pitrou has another opinion?